docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/irsa.md

```diff
@@ -0,0 +1,327 @@
+---
+id: irsa
+title: "IAM roles for service accounts"
+description: "Learn how to configure IAM roles for service accounts (IRSA) within AWS to authenticate workloads."
+---
+
+IAM roles for service accounts (IRSA) is a way within AWS to authenticate workloads in EKS (Kubernetes), for example, to execute signed requests against AWS services. This is a replacement for basic auth and is generally considered more secure.
+
+The following considers the managed services by AWS and provided examples are in Terraform syntax.
+
+## Aurora Postgres
+
+When using the Terraform provider of [AWS](https://registry.terraform.io/providers/hashicorp/aws/latest) with the resource [aws_rds_cluster](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rds_cluster) to create a new rational database (RDS) or Aurora cluster, supply the argument `iam_database_authentication_enabled = true` to enable the IAM roles functionality. See the [AWS documentation](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/UsingWithRDS.IAMDBAuth.html) for availability and limitations.
```

Comment:
I added a section under "troubleshooting" that lists all the versions used within the docs and a disclaimer that it may work with newer versions as well.